charts/streaming-hub/values.yaml

```
@@ -0,0 +1,429 @@
+  # native escape hatch; streamingHub.extraEnvVars carries unmodeled keys.
+  #
+  # DELIBERATELY ABSENT (injected per-Deployment, see the all/ingest/delivery
+  # blocks below): STREAMING_HUB_ROLE, STREAMING_HUB_POSTGRES_MAX_OPEN_CONNS,
+  # STREAMING_HUB_POSTGRES_MAX_IDLE_CONNS, STREAMING_HUB_POSTGRES_CONN_MAX_LIFETIME.
+  # ===========================================================================
+  # -- Native per-key escape hatch (highest precedence). Any UPPER_SNAKE app env
+  #    var can be pinned here verbatim, overriding the grouped field + default.
+  common:
+    configmap: {}
+  # -- Unmodeled extra env vars appended verbatim to the ConfigMap.
+  extraEnvVars: {}
```

YAMLlint (1.37.1)

[error] duplication of key "extraEnvVars" in mapping

(key-duplicates)